lib/usage/index.js

```diff
@@ -21,50 +21,53 @@ export default function (constructor = {}) {
       var parameters = Object.assign(defaults, constructor, options);
       return this.request(`portals/self/usage`, parameters)
       .then(function (response) {
-        return _usage.flattenData(response)
-      })
-      .then(function (data) {
-        let average = data.credits / duration
-        let duration = (data.endTime - data.startTime ) / _usage.periodToMs(data.period)
-        data.activeServices = data.products.length
-        data.average = Math.round(average*100)/100
-        return data
-      })
-      .then(function (data){
-        return _usage.billing()
-        .then(function(response){
-          data.billingCycleStart = response.subscription.termStart
-          data.billingCycleEnd = response.subscription.termEnd
-          data.subscriptionId = response.subscription.smsId
-          data.creditsRemaining = response.subscription.credits - response.subscription.consumedCredits
-          return data
-        })
+        console.log('this is RAW SHIT SUCKAH ', response)
+        return response
+        // return _usage.interperetData(response)
       })
+      // .then(function (data) {
+      //   let average = data.credits / duration
+      //   let duration = (data.endTime - data.startTime ) / _usage.periodToMs(data.period)
+      //   data.activeServices = data.products.length
+      //   data.average = Math.round(average*100)/100
+      //   return data
+      // })
+      // .then(function (data){
+      //   return _usage.billing()
+      //   .then(function(response){
+      //     data.billingCycleStart = response.subscription.termStart
+      //     data.billingCycleEnd = response.subscription.termEnd
+      //     data.subscriptionId = response.subscription.smsId
+      //     data.creditsRemaining = response.subscription.credits - response.subscription.consumedCredits
+      //     return data
+      //   })
+      // })
     },
-
     billing: function (options = {}) {
       return this.request(`/subscription`, options, false, 'https://billing.arcgis.com/sms/rest')
     },
-
-    flattenData: function (response) {
-      var _usage = this
+    interperetData: function (response) {
```

Comment:
Usage API returns things like `count: 8000' and `stype: 'nacfroute'` and other such garbage. This method will turn that into english at some point. Super stubby right now.